docs/rst/tutorials/core/deployment/static_edp/static_edp.rst

```diff
@@ -0,0 +1,231 @@
+.. _tutorials_deployment_static_edp:
+
+Static Discovery
+================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+Background
+----------
+
+`OMG DDS Interoperability specification <https://www.omg.org/spec/DDSI-RTPS/>`_ defines the basic discovery mechanism that allows distributed systems to automatically find and match Endpoints sharing the same Topic.
+This mechanism is performed in two phases as explained in `Fast DDS documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/discovery.html#discovery-phases>`_:
+
+* **Participant Discovery Phase (PDP)**: DDS DomainParticipants discover and acknowledge the existence of remote DomainParticipants.
+  Usually done sending periodic announcements to well-known multicast addresses and ports defined in the specification.
+* **Endpoint Discovery Phase (EDP)**: each DDS DomainParticipant send information about its own endpoints (DataWriters and DataReaders) through the communication channels established in the previous phase.
+
+Standard discovery has its caveats like the required multicast traffic being sent periodically that can saturate the network bandwidth in constraint network architectures with a large number of DDS DomainParticipants.
```

Comment:
```suggestion
Standard discovery has its caveats like the required multicast traffic being sent periodically can saturate the network bandwidth in constraint network architectures with a large number of DDS DomainParticipants.
```

Reply:
I have added some commas to clarify the sentence meaning but I do not think that the word `that` should be removed.